CMakeLists.txt

```diff
@@ -220,6 +219,19 @@ target_compile_definitions      (libtesseract
 )
 set_target_properties           (libtesseract PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS True)
 endif()
+if (Tesseract_USE_OPENCL)
+    target_compile_definitions  (libtesseract PRIVATE USE_OPENCL=1)
+    if (APPLE OR ANDROID)
+        # Inbuilt to NDK
+        find_package(OpenCL REQUIRED)
+        target_link_libraries       (libtesseract ${OpenCL_LIBRARIES})
```

Comment:
This is a full path to library, will make it non-relocatable. Since CMake 3.7 we can use OpenCL::OpenCL:
* https://cmake.org/cmake/help/v3.7/module/FindOpenCL.html